### ISSUE TITLE
Analyse the effort, to reactivate "editing" of submitted datasets in the current Upload Tool implementation

### ISSUE THREAD

**Issue by specka** [zalf-rdm/geonode#153]
## Task Description

Analyse the effort, to reactivate "editing" of submitted datasets in the current Upload Tool implementation; it used to work before, but was disabled for some reasons; First we should finde out
1. Can the editing function be actived? If yes, 
   1.1. it should be done on our test system.
   1.2. What happens, if metadata is changed in the Upload Tool (Redmine?)
   1.3 What happens, if the dataset files are changed / uploaded again? (Redmine?)

2. If not, that we should not enable editing in the Upload Tool for now.

## Additional Information

This question is related to the TF GeoNode meeting from 23.07.2024, in which we discussed general steps of the data publication workflow (https://github.com/orgs/zalf-rdm/discussions/8)

**Comment by specka** [comment]
Evtl. beim Retreat, sonst ein Punkt für später

**Comment by mwallschlaeger** [comment]
I don't think this is in scope anymore. Metadata and data is no longer editable after leaving the upload tool. Authors, data stewards and admins are changing metadata in GN afterwards.

Therefore, i will close this issue now.

@specka @silvadealmeida reopen if I'm wrong
